src/controllers/credentials.ts

```diff
@@ -165,7 +180,35 @@ export class CredentialController {
 		}
 
 		const { credential, policies } = request.body;
-		const verifyStatus = request.query.verifyStatus === 'true' ? true : false;
+		const verifyStatus = request.query.verifyStatus === 'true';
+		const allowDeactivatedDid = request.query.allowDeactivatedDid === "true";
+
+		let issuerDid = "";
+		let decoded: any;
+
+		if (credential.issuer?.id) {
+			issuerDid = credential.issuer.id;
+		} else {
+			try {
```

Comment:
`allowDeactivatedDid` is needed in both credential as well as presentation verify. (Presentation verify might need to resolve multiple DIDs, since the credentials contained might be from different issuers.)

So instead of thinking of this as a helper function specific to credential service, should this not be in did-provider-cheqd? Instead of doing a `resolveDid` in credential service? It could be passed under `verificationOptions` or am i missing something?